### PR TITLE
status: 2023q2: Linuxulator: minor corrections

### PR DESCRIPTION
**Pull request by grahamperrin**

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc [hunk 1]
```diff
@@ -8,12 +8,12 @@ Contact: Dmitry Chagin <dchagin@FreeBSD.org>
 
 The goal of this project is to improve FreeBSD's ability to execute unmodified man:linux[4] binaries.
 
-As of gitref:cbbac5609115[repository=src], preserving of an fpu xsave state across signal delivery on amd64 is implemented.
+As of gitref:cbbac5609115[repository=src], preserving an fpu xsave state across signal delivery on amd64 is implemented.
 That is made possible to run modern golang with preemptive scheduler on.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/linuxulator.adoc:12]: Grammar? I can not quite make sense of this sentence.

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc [hunk 2]
```diff
@@ -8,12 +8,12 @@ Contact: Dmitry Chagin <dchagin@FreeBSD.org>
...
 
 The new facility to specify an alternate ABI root path was added to the man:namei[9].
-Previously, to dynamically reroot lookups, every man:linux[4] syscall where path names translation is needed required a bit of ugly code and used `kern_alternate_path()` which does not properly resolved symlinks with leading `/` in the target.
-For now a non-native ABI (i.e., man:linux[4]) uses one call to `pwd_altroot()` during exec-time into that ABI to specify it's root directory (e.g., [.filename]#/compat/ubuntu#) and forget about path names translation.
+Previously, to dynamically reroot lookups, every man:linux[4] syscall where path names translation is needed required a bit of ugly code and used `kern_alternate_path()` which does not properly resolve symlinks with leading `/` in the target.
+For now a non-native ABI (i.e., man:linux[4]) uses one call to `pwd_altroot()` during exec-time into that ABI to specify its root directory (e.g., [.filename]#/compat/ubuntu#) and forget about path names translation.
 That is made possible to chroot into the Ubuntu compat without having to fix such symlinks by hand.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/linuxulator.adoc:17]: … this sentence, too.

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc (hunk 1 above)
Comment (pauamma) [reply website/content/en/status/report-2023-04-2023-06/linuxulator.adoc:12]: I think what's meant is "That makes it possible to run..."

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc (hunk 2 above)
Comment (pauamma) [reply website/content/en/status/report-2023-04-2023-06/linuxulator.adoc:17]: Same suggested fix.

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc (hunk 1 above)
Comment (lsalvadore) [reply website/content/en/status/report-2023-04-2023-06/linuxulator.adoc:12]: @lemul: Could you please clarify what you meant with this sentence? Thanks.

**Review by lemul — APPROVED** [review]
Im fine with it

**Comment by grahamperrin** [comment]
Sorry, ignore [`f10fb7c` (#191)](https://github.com/freebsd/freebsd-doc/pull/191/commits/f10fb7c7f5823c0bfe878ddfc1e7710c7726e3f0) for now, I thought that this PR for Linuxulator had been completed.

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc (hunk 1 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-04-2023-06/linuxulator.adoc]: Resolved with [`068e91c` (#191)](https://github.com/freebsd/freebsd-doc/pull/191/commits/068e91cfa0a446f63caaf5faae1b1b45ac124087#diff-1b0d1f89ad78dd0ffc75e1b8a3a998c4bd7889d2f2aa426898c53d761112439bR12).

File: website/content/en/status/report-2023-04-2023-06/linuxulator.adoc (hunk 2 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-04-2023-06/linuxulator.adoc]: Resolved with [`068e91c` (#191)](https://github.com/freebsd/freebsd-doc/pull/191/commits/068e91cfa0a446f63caaf5faae1b1b45ac124087#diff-1b0d1f89ad78dd0ffc75e1b8a3a998c4bd7889d2f2aa426898c53d761112439bR17).

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**